### PR TITLE
[Android] Enable transparent TextureView

### PR DESCRIPTION
**Pull request by axinging**
Enable transparent TextureView when user changes the default SurfaceView to TextureView.
Related BUG=XWALK-6519

### CODE REVIEW

**Comment by axinging** [comment]
@halton @lincsoon @kenchris @darktears @fujunwei

File: content/public/android/java/src/org/chromium/content/browser/ContentViewRenderView.java [hunk 1]
```diff
@@ -227,15 +227,24 @@ public ContentReadbackHandler getContentReadbackHandler() {
...
+     * Set the background color of SurfaceView or TextureView.  This method is necessary because the
      * background color of ContentViewRenderView itself is covered by the background of
-     * SurfaceView.
+     * SurfaceView or TextureView.
      * @param color The color of the background.
      */
     public void setSurfaceViewBackgroundColor(int color) {
         if (mSurfaceView != null) {
             mSurfaceView.setBackgroundColor(color);
+            return;
```
Comment (lincsoon) [inline content/public/android/java/src/org/chromium/content/browser/ContentViewRenderView.java]: The `return` here is unnecessary

File: content/public/android/java/src/org/chromium/content/browser/ContentViewRenderView.java [hunk 2]
```diff
@@ -227,15 +227,24 @@ public ContentReadbackHandler getContentReadbackHandler() {
...
+        } else if (mTextureView != null) {
+            mTextureView.setOpaque(isOpaque(color));
+            mTextureView.setBackgroundColor(color);
+            return;
```
Comment (lincsoon) [inline content/public/android/java/src/org/chromium/content/browser/ContentViewRenderView.java]: Ditto

**Comment by axinging** [comment]
Done @lincsoon

**Comment by lincsoon** [comment]
LGTM

**Comment by fujunwei** [comment]
LGTM

**Comment by axinging** [comment]
@rakuco, please help to merge.

**Comment by axinging** [comment]
@rakuco  Ping you again.

**Comment by rakuco** [comment]
merged, thanks. don't forget to roll `DEPS.xwalk`.

**Pull request merged**